--- a/wpa_supplicant/wpa_supplicant.c
+++ b/wpa_supplicant/wpa_supplicant.c
<<<<<<< HEAD
@@ -4248,7 +4248,7 @@ wpa_supplicant_alloc(struct wpa_supplica
=======
@@ -4474,7 +4474,7 @@ wpa_supplicant_alloc(struct wpa_supplica
>>>>>>> 2a18840cc773425668fdfd99429d74ef0ab3a8ef
 	if (wpa_s == NULL)
 		return NULL;
 	wpa_s->scan_req = INITIAL_SCAN_REQ;
-	wpa_s->scan_interval = 5;
+	wpa_s->scan_interval = 1;
 	wpa_s->new_connection = 1;
 	wpa_s->parent = parent ? parent : wpa_s;
 	wpa_s->p2pdev = wpa_s->parent;
